wagtail/admin/templates/wagtailadmin/home/locked_pages.html

```diff
@@ -3,6 +3,7 @@
     <div class="panel nice-padding">{# TODO try moving these classes onto the section tag #}
         <section>
             <h2>{% trans "Your locked pages" %}</h2>
+            <a href="{% url 'wagtailadmin_reports:locked_pages' %}" class="button button-small button-secondary">See all locked pages</a>
```

Comment:
"See all locked pages" should be wrapped with a `{% trans %}` tag. See line above

Reply:
Thanks, I've added in latest.